lib/components/narrative/line-itin/connected-itinerary-body.js

```diff
@@ -61,6 +62,30 @@ class ConnectedItineraryBody extends Component {
     } = this.props
     const { LegIcon } = this.context
 
+    let [pickupAdded, dropOffAdded] = [false, false]
```

Comment:
We don't need an array for these declarations.

Reply:
Ok, separated into their own lines in https://github.com/opentripplanner/otp-react-redux/pull/571/commits/dd2eb27dda8441bac039f6bc0095269df86c09cf